### PR TITLE
add el-4.0 and jsp-3.0 

### PR DESCRIPTION
**Pull request by volosied**
adding jakarata el-4.0 feature & api 

adding jsp-3.0 feature

source: https://github.com/eclipse-ee4j/el-ri/tree/master/api/src/main/java/jakarta/el

### CODE REVIEW

File: dev/com.ibm.websphere.appserver.features/visibility/public/jsp-3.0/com.ibm.websphere.appserver.jsp-3.0.feature [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ com.ibm.websphere.appserver.el-4.0
+-bundles=com.ibm.ws.org.eclipse.jdt.core.3.10.2.v20160712-0000, \
+ com.ibm.websphere.jakartaee.jstl.2.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.servlet:jstl:2.0", \
+ com.ibm.ws.jsp.2.3.jakarta, \
+ com.ibm.ws.jsp.jakarta, \
+ com.ibm.ws.jsp.jstl.facade.jakarta
+-jars=com.ibm.websphere.appserver.spi.jsp; location:=dev/spi/ibm/, \
+ com.ibm.websphere.javaee.jsp.tld.2.2; location:=dev/api/spec/
+-files=dev/spi/ibm/javadoc/com.ibm.websphere.appserver.spi.jsp_1.0-javadoc.zip
+kind=ga
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jsp-3.0/com.ibm.websphere.appserver.jsp-3.0.feature]: Can you make this  kind=noship for now?

File: dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jakarta.jsp-3.0.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jakarta.jsp-3.0
+singleton=true
+-features=com.ibm.websphere.appserver.jakarta.el-4.0; apiJar=false, \
+ com.ibm.websphere.appserver.jakarta.servlet-5.0; apiJar=false
+-bundles=com.ibm.websphere.jakartaee.jsp.3.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.servlet.jsp:jakarta.servlet.jsp-api:3.0.0"
+kind=ga
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jakarta.jsp-3.0.feature]: kind=noship

File: dev/com.ibm.websphere.appserver.features/visibility/public/el-4.0/com.ibm.websphere.appserver.el-4.0.feature [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ org.apache.el;  type="internal", \
+ org.apache.el.lang; type="internal", \
+ org.apache.el.util; type="internal", \
+ org.apache.el.stream; type="internal"
+IBM-ShortName: el-4.0
+Subsystem-Version: 4.0.0
+Subsystem-Name: Expression Language 4.0
+-features=com.ibm.websphere.appserver.jakarta.el-4.0
+-bundles=com.ibm.ws.org.apache.jasper.el.3.0.jakarta
+kind=ga
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/public/el-4.0/com.ibm.websphere.appserver.el-4.0.feature]: kind=noship

**Comment by volosied** [comment]
For issue #11275

**Pull request merged**